src/prompt.ts

```diff
@@ -121,7 +121,7 @@ async function runOrReviseFlow(script: string, key: string) {
   } else if (confirmed) {
     p.outro(`Running: ${script}`);
     console.log('');
-    await execaCommand(script, { stdio: 'inherit' }).catch((err) => {
```

Comment:
awesome

what do you think about even specifying the shell with `shell: process.env.SHELL`, have you looked into the tradeoffs?

I dabbled with this a ways back and _almost_ committed it, but was doing a little more research still but got sidetracked, curious your thoughts on it

Reply:
I think the difference is that shell `shell: true` is hardcoded to bash, whereas `shell: process.env.SHELL` will use the actual current shell, such as zsh